client-src/elements/chromedash-app.js

```diff
@@ -148,6 +150,22 @@ class ChromedashApp extends LitElement {
     });
   }
 
+  fetchPairedUser() {
+    if (this.paired_user !== undefined) {
+      if (this.pageComponent) {
+        this.pageComponent.paired_user = pu;
```

Comment:
is `pu` defined in this context? Should this be `this.paired_user` instead?

Reply:
Whoops.  Fixed.